documentation/restarting.markdown

```diff
@@ -17,13 +17,29 @@ There are several ways to send a HUP signal to the Puppet Server process, but th
 
     kill -HUP `pgrep -f puppet-server`
 
+Starting in version 2.7.0, you can also reload Puppet Server by running the "reload" action via the operating system's service framework.  This is analogous to sending a hangup signal but with the benefit of having the "reload" command pause until the server has been completely reloaded, similar to how the "restart" command pauses until the service process has been fully restarted.
```

Comment:
> This is analogous to sending a hangup signal but with the benefit of having the "reload" command pause until the server has been completely reloaded

Why is this a benefit? I think I know why--it's safer in terms of stability and behaves more consistently--but I'm not sure that the benefit to users is clear here.

Also, if this is the preferred way of doing it, we should either bump the `reload` command process before the `kill -HUP` command, or just remove `kill -HUP` if we really don't want people to use it.